api/v1beta1/flowcollector_types.go

```diff
@@ -280,6 +280,10 @@ type FLPMetrics struct {
 	// ignoreTags is a list of tags to specify which metrics to ignore
 	//+kubebuilder:default:={"egress","packets"}
 	IgnoreTags []string `json:"ignoreTags,omitempty"`
+
+	// disableAlerts is a list of alerts related to FLP that should not be created
```

Comment:
I am not sure what is the best is terms of usage here, a deny-list or an allow-list. Thoughts?
In any case we should document here what are the acceptable values, ie `NetObservNoFlows` and `NetObservLokiError` at the moment

Reply:
I think as a default behavior we want to enable alerts. Using an allow list would make upgrade with new alert a bit more trickier since we would not know if it is still the default list and if we should add the new alert to the upgrade.

I will improve the documentation to mention the possibilities.

Reply:
Also, I would remove the FLP mention, as it might not always be clear to the users when it's related to FLP or not (they don't necessarily know well our architecture)

```suggestion
	// disableAlerts is a list of alerts that should not be created. Possible values are: `NetObservNoFlows`, which is triggered when no flows are being observed for a certain period, and `NetObservLokiError`, which is triggered when flows are being dropped due to Loki errors.
```

Reply:
cc @skrthomas could you please also review the CRD doc added here?
Thanks

Reply:
I added your change but also moved to an enum so we can control the potential values.